src/screens/courses/ActivityCardContainer/index.tsx

```diff
@@ -80,8 +82,10 @@ const ActivityCardContainer = ({
     if (exitConfirmationModal) setExitConfirmationModal(false);
 
     const { profileId } = route.params;
-    if (isCourse) navigation.navigate('CourseProfile', { courseId: profileId, endedActivity: activity?._id });
-    else navigation.navigate('SubProgramProfile', { subProgramId: profileId });
+    if (!isLearner) navigation.navigate('TrainerCourseProfile', { courseId: profileId });
```

Comment:
je pense qu'il faut remplacer `isCourse` par `isLearner`. Et ce que tu appelles ici `is
Learner` c'est `isTrainer`